publish/publish.proj

```diff
@@ -162,7 +162,7 @@
 	       copying them to the indexed directory -->
       <PublishDirectory Condition="'$(WindowsSdkDir)' == ''">$(DownloadDirectory)</PublishDirectory>
       <!-- list of packages that we want to embed symbol signatures in -->
-      <SymbolPackagesToPublishGlob Condition="'$(SymbolPackagesToPublishGlob)' == ''">$(DownloadDirectory)/**/*.nupkg</SymbolPackagesToPublishGlob>
+      <SymbolPackagesToPublishGlob Condition="'$(SymbolPackagesToPublishGlob)' == ''">$(DownloadDirectory)/**/*.symbols.nupkg</SymbolPackagesToPublishGlob>
```

Comment:
In BuildTools, this is used to decide which zips to make signed catalogs for. Nowhere else, according to searches.